.github/workflows/release.yml

```diff
@@ -0,0 +1,76 @@
+#
+# Copyright (c) 2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+name: Release Che Theia
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The version that is going to be released. Should be in format 7.y.z'
+        required: true
+        default: ''
+      forceRecreateTags:
+        description: If true, tags will be recreated. Use with caution
+        required: false
+        default: 'false'
+      pushToNpmJs:
```

Comment:
where it's used?

Reply:
oops, missed some stashed changes, now it's used down the workflow

Reply:
it'll probably never be used, but it's just gonna be there as an option to build image without pushing stuff to npmjs